app/src/main/java/ch/epfl/sdp/db/LeaderboardDAO.java

```diff
@@ -0,0 +1,25 @@
+package ch.epfl.sdp.db;
+
+import androidx.lifecycle.LiveData;
+import androidx.room.Dao;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+
+import java.util.List;
+
+@Dao
+public interface LeaderboardDAO {
```

Comment:
I'm trying to understand, so basically, this is a king of API for sql querries right?

Reply:
Yes, DAO means Data access object.

Reply:
DAO, A mapping of SQL queries to functions. When you use a DAO, you call the methods, and Room takes care of the rest.